library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -36,7 +37,17 @@
 
 	public final String currency;
 
-	public final Double priceValue;
+	public final double priceValue;
```

Comment:
Yes, it can break something if you doing null checking or use some other Double's methods but I think using Object instead primitive type no sense:
1) this value is required and always returned from Play Services
2) this value init only 2 ways - JSONObject.optDouble and Parcel.readDouble. Both variants returned primitive type and no possible to change it because it final